watcher/local/watcher_test.go

```diff
@@ -92,3 +197,109 @@ func newWatcher(t *testing.T, rs channel.RegisterSubscriber) *local.Watcher {
 func makeSignedStateWDummySigs(params *channel.Params, state *channel.State) channel.SignedState {
 	return channel.SignedState{Params: params, State: state}
 }
+
+// randomTxsForSingleCh returns "n" transactions for a random channel.
+func randomTxsForSingleCh(rng *rand.Rand, n int) (*channel.Params, []channel.Transaction) {
+	params, initialState := channeltest.NewRandomParamsAndState(rng, channeltest.WithVersion(0),
+		channeltest.WithNumParts(2), channeltest.WithNumAssets(1), channeltest.WithIsFinal(false), channeltest.WithNumLocked(0))
+
+	txs := make([]channel.Transaction, n)
+	for i := range txs {
+		txs[i] = channel.Transaction{State: initialState.Clone()}
+		txs[i].State.Version = uint64(i)
+	}
+	return params, txs
+}
+
+// trigger is used for triggering events on the mock adjudicator subscription.
+//
+// Adjudicator subscription blocks until it is closed.
+type trigger struct {
```

Comment:
`trigger`  is a bit hard to understand.  
Could you maybe rename it? `adjEventSource` for example.

Reply:
Updated in 4d4d84b.